docs/reference/commandline/daemon.md

```diff
@@ -50,6 +50,7 @@ weight = -1
       --log-driver="json-file"               Default driver for container logs
       --log-opt=[]                           Log driver specific options
       --mtu=0                                Set the containers network MTU
+      --no-color=false                       Prevents the daemon from logging in color
```

Comment:
you'll need to remove the `=false` on this since its not shown any more.
While in in here, can you remove it from any other `docker daemon` option too - like `--help=false` - just the "false by default" ones.